rasa/nlu/utils/__init__.py

```diff
@@ -202,3 +202,41 @@ def json_pickle(file_name: Text, obj: Any) -> None:
 
     with open(file_name, "w", encoding="utf-8") as f:
         f.write(jsonpickle.dumps(obj))
+
+
+def validate_pipeline_yaml(yaml):
+    """Validate NLU pipeline yaml."""
+    from pykwalify.core import Core
+    import pkg_resources
+    from pykwalify.errors import SchemaError
+    from ruamel.yaml import YAMLError
+    from rasa.nlu.config import InvalidConfigError
+    import rasa.utils.io
+    import logging
+
+    log = logging.getLogger("pykwalify")
+    log.setLevel(logging.WARN)
+
+    try:
+        source_data = rasa.utils.io.read_yaml_file(yaml)
+    except YAMLError:
+        raise InvalidConfigError(
+            "The provided config file is invalid. You can use "
+            "http://www.yamllint.com/ to validate the yaml syntax "
+            "of your config file."
+        )
```

Comment:
should we outsource this in a function `validate_yaml(file_path, schema_path)` so that we can reuse this in other parts?